var/ramble/repos/builtin/applications/elk/application.py

```diff
@@ -29,7 +29,12 @@ class Elk(ExecutableApplication):
 
     name = "elk"
 
-    tags("LAPW", "density-functional-theory")
+    tags(
+        "quantum-chemistry",  # Performs quantum chemical calculations
```

Comment:
I think a lot of these comments are just the same thing again and not super useful. Can we make them less verbose somehow?

Reply:
Do you just want all of the comments removed? Or are there specific comments you think are less useful?

I had them added as a way to try and convey to people what the comment should relate to, but they also should be somewhat self explanatory.

Reply:
Some are more useful than others. I'd be OK with all removed, or some more selective middle ground

Reply:
I removed them all, if there are ones you want added back in, let me know.